### PR TITLE
Fix links in test content doc

### PR DESCRIPTION
**Pull request by gkatsev**

### CODE REVIEW

File: docs/test-content.md [hunk 1]
```diff
@@ -38,14 +38,14 @@
 
 Then do *either* of the following:
 
-- Use [Bento4](#bento4) to split the file into an init segment and a fmp4 media segments:
+- Use [Bento4][bento4] to split the file into an init segment and a fmp4 media segments:
```
Comment (gesinger) [inline docs/test-content.md]: Technically I don't think the bento4 links go anywhere.

Comment (gkatsev) [reply docs/test-content.md]: they will point at `[bento4]: https://www.bento4.com/documentation/`

Comment (gesinger) [reply docs/test-content.md]: You're right, I was actually clicking the one on line 31

Comment (gkatsev) [reply docs/test-content.md]: oh, I guess I missed that one, let me update it too

**Pull request merged**